Description: Allow python-keyring newer than 2.0 
 We have newer version than 2.0 on Debian. That's the reason for this change. 
 .
 python-openstackclient (0.3.0-1) unstable; urgency=low
 .
   * Initial release. (Closes: #733557)
Author: gustavo panizzo <gfa@zumbi.com.ar>
Bug-Debian: http://bugs.debian.org/733557

---
Origin: vendor
Forwarded: not-needed
Reviewed-By: gustavo panizzo <gfa@zumbi.com.ar>
Last-Update: <2014-01-06>

--- python-openstackclient-0.3.0.orig/requirements.txt
+++ python-openstackclient-0.3.0/requirements.txt
@@ -1,6 +1,6 @@
 pbr>=0.5.21,<1.0
 cliff>=1.4.3
-keyring>=1.6.1,<2.0
+keyring>=1.6.1
 pycrypto>=2.6
 python-glanceclient>=0.9.0
 python-keystoneclient>=0.4.1
